packages/react-canvas-core/package.json

```diff
@@ -29,7 +29,7 @@
 	"main": "./dist/index.js",
 	"typings": "./dist/@types/index",
 	"dependencies": {
-		"@projectstorm/geometry": "^6.0.0-beta.1"
```

Comment:
This: `^6.0.0-beta.1` is actually pulling down version `6.0.0-y.1` from our registry and it's causing us problems. I had to lock the version number down get around this as we are unable to remove the rogue version